addon/lib/toolbar-button.js

```diff
@@ -23,11 +26,22 @@ let settings;
 let button;
 let panel;
 let collapsed;
+let prefs;
 
-function setActionButton(dark) {
-  const iconPrefix = dark ? './icon-inverted' : './icon';
+function themeChanged() {
+  setTimeout(setButton);
```

Comment:
weirdly, without the `setTimeout` it sometimes (always?) picks up the style *before* the change is applied